tools/probes/common/check_ruciomover

```diff
@@ -0,0 +1,258 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Authors:
+# - Nicolo Magini, <Nicolo.Magini@cern.ch>, 2017
+
+'''
+Probe to check read/write with all protocols on target RSE with rucio client
+'''
+
+import os
+import sys
+import re
+import tempfile
+import time
+import uuid
+from commands import getstatusoutput
+import filecmp
+
+import argparse
+
+from rucio.common.config import config_get
+from rucio.common.exception import RSEProtocolNotSupported, RSENotFound
+from rucio.rse import rsemanager
+from rucio.client.rseclient import RSEClient
+from rucio.client.replicaclient import ReplicaClient
+
+def upload(rse, protocol, scope, lfn, no_register=False):
+        """
+        Use the rucio upload command to stage out the file.
+        :param rse - destination RSE
+        :param protocol - transfer protocol
+        :param scope - file scope      
+        :param lfn - file lfn
+	:param no_register - skip file registration in Rucio; also disables auto-deletion at lifetime expiration
+        """
+
+	if no_register:
+		no_reg_option='--no-register'
+	else:
+		no_reg_option='--lifetime 3600'
+
+        cmd = 'rucio upload %s --rse %s --protocol %s --scope %s %s' % (no_reg_option, rse, protocol, scope, lfn)
+
+        print 'Will use the following command for stageOut:\n%s\n' % cmd
+        s, o = getstatusoutput(cmd)
+        print 'stageOut output:\n%s\n' % o
+        
+        if s:
+            raise Exception('stageOut failed -- rucio upload did not succeed: %s' % o.replace('\n', ''))
+
+
+def download(rse, protocol, scope, lfn, dst, pfn=None):
+        """
+        Use the rucio download command to stage in the file.
+        :param rse:   source RSE
+        :param protocol: transfer protocol
+        :param scope: file scope
+        :param lfn:   file lfn
+        :param dst:   local destination directory
+	:param pfn:   file pfn (required when the downloaded file is not registered in Rucio
+        """
+	pfnOption=''
+	if pfn:
+		pfnOption='--pfn %s' % pfn
+
+        cmd = 'rucio download --dir %s --rse %s --protocol %s %s %s:%s' % (dst,
+                                                                         rse,
+                                                                         protocol,
+									 pfnOption,
+                                                                         scope,
+                                                                         lfn)
+        print 'Will use the following command for stageIn:\n%s\n' % cmd
+        s, o = getstatusoutput(cmd)
+        
+        print 'stageIn output:\n%s\n' % o
+
+        if s:
+            raise Exception('stageIn failed -- rucio download did not succeed: %s' % o.replace('\n', ''))
+
+
+OK, WARNING, CRITICAL, UNKNOWN = 0, 1, 2, 3
+
+parser=argparse.ArgumentParser()
```

Comment:
hmmm pylint should have given you "missing spaces around operator" warning here. did you run a recent version of pylint against the file?